README.md

```diff
@@ -0,0 +1,90 @@
+# Rockets-React-Redux
+A Space Travelers' Hub consisting of Rockets, Missions, and a Profile section.
+
+![screenshot](/ScreenShot.PNG)
+
+## Built With
+
+- React
+- JSX
+- Redux
+
+## Live Demo
+
+- [Live Demo Link]( https://rocket-react-hub.netlify.app/)
```

Comment:
- [ ] As stated in the project requirements, kindly record a video discussing the aspects listed in the requirements.

Reply:
Hi @mikenath223 

Thanks for your speedy response.
However, I do not think the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md) stated we add a video.
Kindly review the requirements and guide us further.